ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/ontologies/OntologiesView.java

```diff
@@ -50,10 +50,11 @@
                                         RefreshPreviewButtonClicked.HasRefreshPreviewButtonClickedHandlers,
                                         RestoreAppButtonClicked.HasRestoreAppButtonClickedHandlers {
 
-    enum TreeType {
+    enum ViewType {
         ALL, EDITOR, PREVIEW
     }
 
+    String TRASH_CATEGORY = "Trash";
```

Comment:
Aren't interface variables static and final already?  When I copy-pasted it here, IntelliJ complained that they were redundant too.